examples/lfric/eg4/solver_mod.x90

```diff
@@ -107,8 +107,9 @@ subroutine jacobi_solver_algorithm(lhs, rhs, mm, mesh, n_iter)
   implicit none
 
   integer(i_def),      intent(in)    :: n_iter
-  type(field_type),    intent(inout) :: lhs, rhs
-  type(operator_type), intent(inout) :: mm
+  type(field_type),    intent(inout) :: lhs
```

Comment:
Please could you explain why this is `inout` and not `in`?

Reply:
Because of the LFRic code equivalents where particular field intents had to be left as ``inout`` for successful compilation.